### DISCUSSION TITLE
建议过期删除功能的定义粒度能支持到超表级别

### DISCUSSION THREAD

**Discussion by leoasc** [taosdata/TDengine#8531] [Ideas]
让超表默认继承数据库的过期删除参数，但是可以单独定义。
设备数据有高频次的，也有低频次的，如果是统一的过期删除时间的话，低频数据可能会被全部删除而查不到数据了。
高低频数据分库可以解决这个问题，但是会带来存取判断计算的开销，增加应用软件设计的复杂度。

**Comment by yu285** [comment]
我们删除数据是按照数据时间戳的时间和系统时间的差值，不是按照数据入库后的保留时间来删除

这样不论高频还是低频，都是一样的，高频数据10万条，低频数据1条，只要时间戳没有超出keep，都不会被清理，
